### ISSUE TITLE
Extension controls should be next to where the extensions info is rendered in focus order

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#101185]
CC: @isidorn

Issue Type: <b>Bug</b>

1. Open the info for any extension. E.g, ctrl+shift+x, and go to the first result for "Code Spell Checker".

Expected: when the problems view is open at the bottom, pressing shift+tab from the top of the virtual document which renders extension info should cause focus to move to the controls for managing the extension. E.g, the view changelog and install / uninstall extension button.

Actual: I'm forced to tab through all the controls in the problems pane first, which is about 8 tab stops before getting to where I need to be to install the extension

As an aside, pressing f6 does allow you to move focus to the virtual document where the extension's info is rendered. But f6 / shift+f6 isn't able to move focus from it, so focus is trapped there when using f6 / shift + f6 to cycle through major parts of the UI. Not sure if I should open a separate issue for this.

VS Code version: Code - Insiders 1.47.0-insider (9bf6af04dcb5c2e531cedca83cd188939ca1b0db, 2020-06-26T18:55:07.038Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.86GB (4.37GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|17%|
</details><details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
notes|can|1.0.1
vscode-markdownlint|Dav|0.36.1
vscode-eslint|dba|2.1.5
gitlens|eam|10.2.2
prettier-vscode|esb|5.1.0
Go|gol|0.14.4
rest-client|hum|0.24.1
python|ms-|2020.6.90262
remote-wsl|ms-|0.44.4
cpptools|ms-|0.28.3
vsnotes|pat|0.7.1
rust|rus|0.7.8
markdown-preview-enhanced|shd|0.5.12
code-spell-checker|str|1.9.0
vscode-webhint|web|1.5.6

</details>

**Comment by sandy081** [comment]
> I'm forced to tab through all the controls in the problems pane first, which is about 8 tab stops before getting to where I need to be to install the extension

**Comment by isidorn** [comment]
@sandy081 here are two issues from the comment above

1. Have extension descriptoin open and focused in a web view, and have the panel open. Press Shit + Tab -> focus goes to panel, it should go to the sidebar
2. Have extension description open and focused. Press F6 -> no effect, focus is not moved across the workbench

Notice how the F6 key is registered with no precondition [here](https://github.com/microsoft/vscode/blob/b58df64fd23e9639972d4b669d37365a76955d07/src/vs/workbench/browser/actions/navigationActions.ts#L267) so I am not sure why the extension webview is eating up the `F6` and `shift + F6` key.

**Comment by sandy081** [comment]
@mjbvz Can you please let me know if there are any web view options I can use to fix this?

**Comment by mjbvz** [comment]
@sandy081 I pushed a fix for F6.

I'm not sure about the expected shift+tab behavior. Can you please test out the next insiders build and let me know what should be happening?

**Comment by sandy081** [comment]
F6 is working but `shift+tab` from web view editor is going to panel. It is also true when you open Markdown preview editor and do shift+tab.

**Comment by isidorn** [comment]
@mjbvz can we change that the shift + tab from webviews goes to sidebar and not to the panel. That is the only remaining problem here it seems?

**Comment by isidorn** [comment]
@mjbvz friendly ping for this since I know you were fixing some webview focus issues lately

**Comment by isidorn** [comment]
@mjbvz friendly ping

**Comment by mjbvz** [comment]
I'm not sure how to fix this
